File: JANUARY/05-01-2026/1975.java
class Solution {
    public long maxMatrixSum(int[][] matrix) {
        long sum = 0;
        int neg = 0;
        int mn = Integer.MAX_VALUE;
        for (int[] r : matrix) {
            for (int x : r) {
                if (x < 0) neg++;
                sum += Math.abs((long)x);
                mn = Math.min(mn, Math.abs(x));
            }
        }
        if ((neg & 1) == 1) sum -= 2L * mn;
        return sum;
    }
}

File: JANUARY/05-01-2026/1975.kt
class Solution {
    fun maxMatrixSum(matrix: Array<IntArray>): Long {
        var sum = 0L
        var neg = 0
        var mn = Long.MAX_VALUE
        for (row in matrix) {
            for (x in row) {
                if (x < 0) neg++
                val ax = kotlin.math.abs(x.toLong())
                sum += ax
                if (ax < mn) mn = ax
            }
        }
        if (neg % 2 == 1) sum -= 2 * mn
        return sum
    }
}